tests/test_launch_methods/test_jsrun/jobs.sh

```diff
@@ -0,0 +1,79 @@
+#!/bin/bash
+
+# Begin LSF Directives
+#BSUB -P BIP178
+#BSUB -W 0:30
+#BSUB -nnodes 2
+#BSUB -J jsrun_test
+#BSUB -o jsrun_test.%J
+#BSUB -e jsrun_test.%J
+
+# CONFIG
+path=`pwd`
+d=`date +%D | sed 's/\//_/g'`
+host=`/bin/hostname`
+dest=$RCT_TESTS_EMAIL       # To be set in bashrc
+
+run_cmd() {
+    res_set=$1
+    omp_env=$2
+    cmd=$3
+    exp_out=$4
+    tid=$5
+
+    # Create required folders
+    rm results_$d/$tid -rf
+    mkdir results_$d/$tid
+
+    # Record resource file, omp env, and cmd in log
+    echo $res_set >> results_$d/$tid/full_output.log
+    echo $omp_env >> results_$d/$tid/full_output.log
+    echo $cmd >> results_$d/$tid/full_output.log
+
+    # Set omp env, create resource file and run cmd
+    $omp_env
+    cmd=`echo $cmd | sed "s#resource-file#$path/$tid/res_set#g"`
+    $cmd >> results_$d/$tid/jsrun_output.log
+
+    # Jsrun output to full output log file
+    echo "Output:" >> results_$d/$tid/full_output.log
+    cat results_$d/$tid/jsrun_output.log >> results_$d/$tid/full_output.log
+}
+
+assert() {
+    actual=$1
+    expected=$2
+    tid=$3
+
+    cp $actual results_$d/$tid/jsrun_output_dup.log
+    # Remove Node ID from actual output
+    sed -i 's/Node ....../Node /g' results_$d/$tid/jsrun_output_dup.log
+    if ! diff $expected results_$d/$tid/jsrun_output_dup.log; then
+        op="$tid failed: Output location = results_$d/$tid/jsrun_output.log, Diff location = results_$d/$tid/full_output.log"
+        echo "Diff between expected and actual (bar node ID):" >> results_$d/$tid/full_output.log
+        diff $expected results_$d/$tid/jsrun_output_dup.log >> results_$d/$tid/full_output.log
```

Comment:
Can you put the absolute path here? I am not sure from the email where the log file is. As a result, I cannot check it

Reply:
@vivek-bala any follow up on this?